ci-operator/jobs/jws-qe/interop-ocp-ci/jws-qe-interop-ocp-ci-main-periodics.yaml

```diff
@@ -637,3 +637,84 @@ periodics:
     - name: result-aggregator
       secret:
         secretName: result-aggregator
+- agent: kubernetes
+  cluster: build03
+  cron: 0 6 25 12 *
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+  extra_refs:
+  - base_ref: main
+    org: jws-qe
+    repo: interop-ocp-ci
+  labels:
+    ci-operator.openshift.io/cloud: aws
+    ci-operator.openshift.io/cloud-cluster-profile: aws-cspi-qe
+    ci-operator.openshift.io/variant: ocp-4.17-lp-interop
+    ci.openshift.io/generator: prowgen
+    job-release: "4.17"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-jws-qe-interop-ocp-ci-main-ocp-4.17-lp-interop-jws-interop-aws-fips
```

Comment:
Either need to create `.config.prowgen` file and add `slack_reporter` dictionary; or manually add `slack_reporter` config to the jobs file here.